sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -22,6 +22,13 @@
 
 def load_media_settings():
     global g_dict
+
+    version_info = device_info.get_sonic_version_info()
+    if version_info and version_info['asic_type'] == 'mellanox':
```

Comment:
@tshalvi NO platform specific checks allowed . Xcvrd code should be platform independent.